config/initializers/pusher.rb

```diff
@@ -4,6 +4,6 @@
   Pusher.app_id = ENV['PUSHER_APP_ID']
   Pusher.key = ENV['PUSHER_APP_KEY']
   Pusher.secret = ENV['PUSHER_APP_SECRET']
-  Pusher.host = ENV['PUSHER_HOST'] if ENV['PUSHER_HOST']
+  Pusher.host = ENV['PUSHER_HOST'].dup if ENV['PUSHER_HOST']
```

Comment:
What is the reason for this `#dup` call?

Reply:
The `dup` call returns a string with the same value without the metadata, like frozen as an example. When we get values from an environment variable it is frozen as default, but in the application, the `pusher` send this string to `newrelic_rpm` that tries to update the variable, so if it is frozen an error will be raised, but if I use the method `dup`, the string is no more frozen and the error doesn't occur.

Reply:
Got it.